### ISSUE TITLE
[BUG] Player doesn't prevent sleep anymore

### ISSUE THREAD

**Issue by ngelus** [DATL4G/BurningSeries-Android#53]
**Describe the bug**
The video / Stream Player usually prevented my phone from turning off the screen and going to sleep. In this newest version it doesn't do this anymore.

**To Reproduce**
1. Go to any episode of any series.
2. Play the episode.
3. Wait for the time you've defined as the timeout for the display.
4. See the display getting darker and turning off.

**Expected behavior**
The player should prevent the display from turning off.

**Smartphone (please complete the following information):**
 - Device: OnePlus 9
 - OS: Android 13
 - Version: 4.7.0 (latest release)

**Additional context**
I'm going to revert to the last version for now.

**Comment by DatL4g** [comment]
This part didn't change at all and doesn't happen on my devices.
Did you do a System Update lately?
Could you try again after clearing the cache

**Comment by ngelus** [comment]
Okay. I'm gonna upgrade to v4.7.0 again, clear the cache and try again now.

**Comment by ngelus** [comment]
Didn't do any updates recently. At least I didn't receive a message that it's been updated like I usually do. My device updates automatically in the night.

**Comment by ngelus** [comment]
Oh... Tested and now it works fine xD
I gotta say that I've experienced this multiple times back on the previous version of the app aswell tho.
Mostly when I pause the player, close it to background, then open the player back up again and resume. I also had the same problem when pausing, putting it in PiP, restoring it to full screen and resuming. I don't know if this is still worth checking out.

Btw; I had the display timeout set to the minimum of 15 seconds. I have now increased it to 30 seconds. I don't know if that might have something to do with it, so I'm just leaving this information here.
